DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreWebsiteDetails.cshtml

```diff
@@ -12,23 +11,13 @@
 }
 
 @section NavMenuItems {
-  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml"/>
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
-    @if (errorHasOccurred)
-    {
-      <vc:error-summary order-of-property-names="@(new[] {
-                                                   nameof(Model.CentreTelephone),
-                                                   nameof(Model.CentreEmail),
-                                                   nameof(Model.CentrePostcode),
-                                                   nameof(Model.OpeningHours),
-                                                   nameof(Model.CentreWebAddress),
-                                                   nameof(Model.OrganisationsCovered),
-                                                   nameof(Model.TrainingVenues),
-                                                   nameof(Model.OtherInformation)
-                                                 })" />
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.CentreTelephone), nameof(Model.CentreEmail), nameof(Model.CentrePostcode), nameof(Model.ShowCentreOnMap), nameof(Model.OpeningHours), nameof(Model.CentreWebAddress), nameof(Model.OrganisationsCovered), nameof(Model.TrainingVenues), nameof(Model.OtherInformation) })"/>

```

Comment:
Can you sort out the formatting here. Looks like the auto formatter doesn't like this setup.